newrelic.yml

```diff
@@ -692,6 +692,34 @@ common: &default_settings
   # If true, the agent automatically detects that it is running in a Pivotal Cloud Foundry environment.
   # utilization.detect_pcf: true
 
+  # BEGIN security agent
```

Comment:
This might be a good place to put instructions about how to use the security agent (ex. only in a dedicated test environment, it will delete your data, etc.)

Reply:
Good call. I think I can bring this to the cross agent spec discussion as well.